templates/data/index.html

```diff
@@ -0,0 +1,431 @@
+
+					<div class="row">
+						<hr class="p-rule">
+						<div class="col-3">
+							<h3 class="p-muted-heading">What's included</h3>
+						</div>
+						<div class="col-9">
+							<div class="p-section--shallow">
+								<hr class="p-rule is-muted">
+								<ul class="p-list--divided">
+									<li class="p-list__item has-bullet">Production-grade Data Fabric deployment on cloud IaaS or Kubernetes</li>
+									<li class="p-list__item has-bullet">Workshop</li>
+									<li class="p-list__item has-bullet">Operating system tuning</li>
+									<li class="p-list__item has-bullet">Storage integration</li>
+									<li class="p-list__item has-bullet">Design document</li>
+									<li class="p-list__item has-bullet">Deployment guide</li>
+									<li class="p-list__item has-bullet">Knowledge transfer on operating the Data Fabric solution with Canonical tooling</li>
+								</ul>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+		<br class="u-hide--large u-hide--medium" />
+		<div class="col">
+			<hr class="p-rule--highlight">
+			<div class="p-card" style="height: 100%;">
+				<div>
+				<h3 class="p-section">Fully-managed Data Fabric operations</h3>
+			
+				<div class="p-card__content">
+					<p>Full remote operations of your Data Fabric solution. Ideal to accelerate time-to-market. Most cost-effective approach versus PaaS.</p>
+					<p>We manage the data platform for you, on the cloud or in your data centre.</p>
+					<hr class="p-rule">
+					<h3 class="u-align-text--right">&dollar;3,099<br /><small>per VM per year</small></h3>
+
+					<div class="row">
+						<hr class="p-rule">
+						<div class="col-3">
+							<h3 class="p-muted-heading">What's included</h3>
+						</div>
+						<div class="col-9">
+							<div class="p-section--shallow">
+								<hr class="p-rule is-muted">
+								<ul class="p-list--divided">
+									<li class="p-list__item has-bullet">Logging, monitoring and alerting</li>
+									<li class="p-list__item has-bullet">24/7 operations with fully staffed Operations Centre</li>
+									<li class="p-list__item has-bullet">Proactive management</li>
+									<li class="p-list__item has-bullet">Incident and problem resolution</li>
+									<li class="p-list__item has-bullet">Software updates</li>
+									<li class="p-list__item has-bullet">SLA</li>
+								</ul>
+							</div>
+						</div>
+					</div>
+					
+				</div>
+				</div>
+				<div>
+				<p>
+					<a href="http://ubuntu.com/managed/apps">Learn about fully-managed&nbsp;&rsaquo;</a>
+				</p>
+			</div>
+			</div>
+		</div>
+	</div>
+</section>
+
+<section class="p-strip--light is-deep">
+  <div class="row">
+    <div class="col-9">
+      <p class="p-heading--2">Tell us your story and talk about your needs with our experts.</p>
+			<div class="p-block">
+				<h2>
+					<a class="js-invoke-modal" href="https://ubuntu.com/openstack/contact-us?product=openstack">Get in touch&nbsp;&rsaquo;</a>
```

Comment:
Why would this lead to openstack contact us?

Reply:
This is what's currently there, I guess this can also be replaced once we get c.com/data page released.

Reply:
Fixed after rebase.